rest-server/data/ur.json

```diff
@@ -3,21 +3,22 @@
   "engineFactory": "com.actionml.engines.ur.UREngine",
   "sparkConf": {
     "master": "yarn",
+    "appName": "harness-spark-test-app",
+    "mongo.database": "harness_meta_store",
+    "mongo.collection": "engines",
```

Comment:
same comments as `mongo.database` above

Reply:
Why is the `harness_meta_store` name needed? This should never be used outside the administrator.